eZ/Publish/Core/Persistence/Legacy/Tests/Content/UrlAlias/UrlAliasHandlerTest.php

```diff
@@ -1170,6 +1170,76 @@ public function testPublishUrlAliasCreatesUniqueAlias()
         );
     }
 
+    /**
+     * Test for the publishUrlAliasForLocation() method.
+     *
+     * Japanese text is not transliterated, and would result in empty string if numbers were not added.
+     *
+     * @covers \eZ\Publish\Core\Persistence\Legacy\Content\UrlAlias\Handler::publishUrlAliasForLocation
+     * @depends testPublishUrlAliasForLocation
+     * @group publish
+     */
+    public function testPublishUrlAliasCreatesNonEmptyAlias()
```

Comment:
`eZ/Publish/Core/Persistence/Legacy/Tests` namespace is not a good place for tests which operate on database. It means that it will be tested on sqlite only. Such tests are integration tests and should be done in e.g. `\eZ\Publish\API\Repository\Tests\URLAliasServiceTest`. Alternatively you could change it into unit test with mocks.

_Yes, I know you've copy-pasted this from other tests like this, but this didn't change the fact it was oversight by someone who started writing these tests. Sorry :(_

Reply:
@alongosz Thanks (no really, I mean it - this ought to be properly tested in all dbs :) )

Reply:
Removed test, added test in `\eZ\Publish\API\Repository\Tests\URLAliasServiceTest`, verified locally that it fails before the test and passes after.